File: sample/src/main/java/com/adzerk/android/sdk/sample/RoundedAvatarProvider.java
package com.adzerk.android.sdk.sample;

import android.annotation.TargetApi;
import android.graphics.Outline;
import android.view.View;
import android.view.ViewOutlineProvider;

@TargetApi(21)
public class RoundedAvatarProvider extends ViewOutlineProvider {
    @Override
    public void getOutline(View view, Outline outline) {
        outline.setOval(0, 0, view.getWidth(), view.getHeight());
    }
}

File: sample-kt/src/main/java/com/adzerk/android/sdk/sample2/RoundedAvatarProvider.kt
package com.adzerk.android.sdk.sample2

import android.annotation.TargetApi
import android.graphics.Outline
import android.view.View
import android.view.ViewOutlineProvider

@TargetApi(21)
class RoundedAvatarProvider : ViewOutlineProvider() {
    override fun getOutline(view: View, outline: Outline) {
        outline.setOval(0, 0, view.width, view.height)
    }
}
